docs/website/docs/general-usage/schema.md

```diff
@@ -227,9 +227,6 @@ You are able to bring your own `row_key` by adding a `_dlt_id` column/field to y
 
 `merge` write disposition requires an additional nested reference that goes from **nested** to **root** table, skipping all parent tables in between. This reference is created by [adding a column with a hint](incremental-loading.md#forcing-root-key-propagation) `root_key` (named `_dlt_root_id` by default) to nested tables.
 
-### Table references
-You can annotate tables with table references. This feature is coming soon.
```

Comment:
@joscha thanks for the PR. The section above describes nested references, not table references, if you read the whole references section you can see that this differentiation is done at the beginning of the section. We have a conceptual implementation of these table references but they are not available in the core library yet.

Reply:
I see - should this be `####` then, to be one level below?

Reply:
changed it in 3aaa33835c90cfddecc3a5414ba2e32ba2f2123c

Reply:
no, not really, ### is correct.

Reply:
OK, I don't think it makes sense:

![2024-12-18_10-52](https://github.com/user-attachments/assets/cd5ea7e1-ee21-49d9-a0d1-a941b6911923)